README.md

```diff
@@ -43,6 +43,7 @@ Following parameters are available to customize the elastic cluster:
 - [storage](https://kubernetes.io/docs/user-guide/persistent-volumes/)
   - type: Defines the type of storage to provision based upon cloud (e.g. `gp2`)
   - storage-class-provisioner: Defines which type of provisioner to use (e.g. `kubernetes.io/aws-ebs`)
+  - storage-class: Defines the name of an existing StorageClass object to use
```

Comment:
Could you add a bit about how this is optional? Or something about it's use case, I don't want to confuse folks to think this is required, but I understand there are use-cases.